### PR TITLE
Providing Typedoc and Sassdoc references for EN and JP topics.

### PR DESCRIPTION
**Pull request by HristoP96**
Providing Typedoc references for all the feature/components/components-members in the below listed topics:

Cell component
Column group (Multi column headers)
Column component
Grid summary
Grid crud
Pagination
Sorting
Linear Progress bar
Circular Progress bar
Dialog
Slider
Toggle
Layout Manager
Carousel
Calendar
DatePicker

### CODE REVIEW

**Comment by zdrawku** [comment]
@HristoP96 Use this PR to add all changes related to this task

File: en/components/grid_multi_column_headers.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
 <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="grid-multi-column-headers-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
 </div>
 
-The declaration of `Multi column header` could be achieved by wrapping a set of columns into `igx-column-group` component with `header` title passed.
+The declaration of `Multi column header` could be achieved by wrapping a set of columns into [`igx-column-group`]({environment:angularApiUrl}/classes/igxcolumngroupcomponent.html) component with [`header`]({environment:angularApiUrl}/classes/igxcolumngroupcomponent.html#header) title passed.
```
Comment (hanastasov) [inline en/components/grid_multi_column_headers.md]: `header` is linked to columngroupcomponent here

File: en/components/layout.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
 <div class="divider--half"></div>
 
 ###Usage
-Use the **igxLayout** directive on a container element to specify the layout
+Use the [**igxLayout**]({environment:angularApiUrl}/classes/igxlayoutdirective.html) directive on a container element to specify the layout
 direction for its children: horizontally with `igxLayoutDir="row"` or vertically with
 `igxLayoutDir="column"`.
```
Comment (hanastasov) [inline en/components/layout.md]: igxLayoutDir should be linked to /igxlayoutdirective.html#dir

File: en/components/linear_progress.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ export class AppModule {}
 ```
 
 And now let's create a simple example in which we are going to simulate a continuous process, that is triggered on button click. In addition you can notice
-that the speed of loading depends on the `max` attribute that we have set to our `igx-linear-bar`. In the first case our max value is set to 200 and loads slower than in the second where it is set to 100 (default value). So let's see the code...
+that the speed of loading depends on the [`max`]({environment:angularApiUrl}/classes/igxlinearprogressbarcomponent.html#max) attribute that we have set to our `igx-linear-bar`. In the first case our max value is set to 200 and loads slower than in the second where it is set to 100 (default value). So let's see the code...
```
Comment (hanastasov) [inline en/components/linear_progress.md]: each mention of `max` property should be linked

**Pull request merged**